dev/build_release.py

```diff
@@ -287,11 +288,32 @@ def start_subsystem_build(self, name):
     # Note: 'candidate' is just the gradle task name. It doesn't indicate
     # 'release candidate' status for the artifacts created through this build.
     return BackgroundProcess.spawn(
-      'Building and publishing {name}...'.format(name=name),
+      'Building and publishing Debian for {name}...'.format(name=name),
       'cd "{gradle_root}"; ./gradlew {extra} {target}'.format(
           gradle_root=gradle_root, extra=' '.join(extra_args), target=target)
     )
 
+  def start_container_build(self, name):
+    """Start a subprocess to build a container image of the subsystem.
+
+    Uses Google Container Builder with configuration files produced
+    during BOM generation to build the container images. The configuration
+    files are assumed to be in the parent directory of the subsystem's
+    Gradle root.
+
+    Args:
+      name [string]: Name of the subsystem repository.
+
+    Returns:
+      BackgroundProcess
+    """
+    gradle_root = self.determine_gradle_root(name)
+    return BackgroundProcess.spawn(
+      'Building and publishing container image for {name}...'.format(name=name),
```

Comment:
If the other registries follow the pattern of `${provider}/${project}/${repo}:${tag}` then I'm all for adding the registry as an argument to this script. Also I believe we'd need a `cloud-builder/docker` for the registry provider as well: https://github.com/jtk54/spinnaker/blob/e983cc878419f3ca64bc48ca4eaf8ec56ff52b9b/dev/generate_bom.py#L38 -- I'll check which providers are supported with the Container Builder team.